catalog/urls.py

```diff
@@ -4,6 +4,6 @@
 app_name = 'catalog'
 
 urlpatterns = [
-    path('home_html/', views.home_html, name='home_html'),
-    path('contacts_post/', views.contacts_post, name='contacts_post'),
+    path('home/', views.home_html, name='home_html'),
```

Comment:
В атрибуте name указываем не название вьюшки, а просто название контроллера, по которому можно будет к нему обращаться. Поэтому в данном случае можно написать например так:
```suggestion
    path('/', views.home_html, name='home-page'),
```